### PR TITLE
[WIP] Audited

### PR DESCRIPTION
**Pull request by jake3030**

### CODE REVIEW

File: app/models/airtable_user.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Airrecord.api_key = ENV['AIRTABLE_API_KEY']
 
 class AirtableUser < Airrecord::Table
+  audited
```
Comment (jake3030) [inline app/models/airtable_user.rb]: I dont think this will work since this isnt on our DB, it's on AirTable.

File: app/models/project.rb [hunk 2]
```diff
@@ -2,4 +2,11 @@ class Project < ApplicationRecord
   belongs_to :lead, class_name: "User"
   has_and_belongs_to_many :volunteers, class_name: "User", join_table: "projects_volunteers"
   has_and_belongs_to_many :skills
+
+  audited
+
+  audited associated_with: :user
+  audited associated_with :skills
```
Comment (jake3030) [inline app/models/project.rb]: should this be `associated_with:`

File: app/models/skill.rb [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
   has_and_belongs_to_many :users
-  
+
   scope :tech_skills, -> () { where(tech: true).order("name ASC") }
   scope :non_tech_skills, -> () { where(tech: false).order("name ASC") }
 
+  audited
+
+  audited associated_with :project
+  audited associated_with :user
```
Comment (jake3030) [inline app/models/skill.rb]: should these be `associated_with:`?

File: app/models/airtable_user.rb (hunk 1 above)
Comment (sdklos) [reply app/models/airtable_user.rb]: Yeah I wasn't sure if that was necessary. I was wondering if audited might track the differencez between each sync action

**Pull request merged**